lua/feline/providers/file.lua

```diff
@@ -61,6 +61,22 @@ local function get_unique_filename(filename, shorten)
     return string.reverse(string.sub(filename, 1, index))
 end
 
+-- Get path separator depending on OS
+local function get_path_separator()
+    if jit then
+        local os = string.lower(jit.os)
+        if os == 'linux' or os == 'osx' or os == 'bsd' then
+            return '/'
+        else
+            return '\\'
```

Comment:
```suggestion
            return [[/]]
        else
            return [[\]]
```

Raw strings look cleaner in this case imo